pkg/rc/replication_controller.go

```diff
@@ -264,6 +265,26 @@ func (rc *replicationController) WatchDesires(quit <-chan struct{}) <-chan error
 
 func (rc *replicationController) meetDesires(rcFields fields.RC) error {
 	rc.logger.NoFields().Infof("Handling RC update: desired replicas %d, disabled %v", rcFields.ReplicasDesired, rcFields.Disabled)
+	podID := rcFields.Manifest.ID()
+	launchableStanzas = rcFields.Manifest.GetLaunchableStanzas()
+	for launchableID, launchableStanza := range launchableStanzas {
+		artifactUrl, _, err := rc.artifactRegistry.LocationDataForLaunchable(podID, launchableID, launchableStanza)
+		if err != nil {
+			rc.logger.WithError(err).Errorln("Unable to retrieve location for launchable")
+		}
+		res, err := http.Get(artifactUrl)
```

Comment:
Would a HEAD request suffice? Artifacts can potentially be gigabytes in size, so it would be a huge advantage to use HEAD.

Reply:
When I try HEAD, it always gives me a 200 response whether the artifact url is valid or not.

Reply:
I'll try debugging the head request, I agree it should not be GET.